deploy/examples/csi/nfs/pod.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: csinfs-demo-pod
```

Comment:
nfs-demo-pod

Reply:
Both the following have `csi` prefix :sweat_smile: 
https://github.com/rook/rook/blob/b80e922a7f75d7cdce76cb347b46732a4c785de3/deploy/examples/csi/rbd/pod.yaml#L5
https://github.com/rook/rook/blob/b80e922a7f75d7cdce76cb347b46732a4c785de3/deploy/examples/csi/cephfs/pod.yaml#L5